start

```diff
@@ -33,16 +37,39 @@ done
 
 cd "${projectRoot}"
 
-./ffc-sfd-landing-page/scripts/start -d;
-./ffc-sfd-auth/scripts/start -d;
-./ffc-sfd-business/scripts/start -d;
-./ffc-sfd-data/scripts/start -d;
-./ffc-sfd-permissions/scripts/start -d;
-./ffc-sfd-permissions-api/scripts/start -d;
-./ffc-sfd-ahwp-stub/scripts/start -d;
-./ffc-sfd-messages/scripts/start -d;
-./ffc-sfd-messages-processor/scripts/start -d;
-./ffc-sfd-customer-receiver-messages/scripts/start -d;
-./ffc-sfd-customer-receiver-payments/scripts/start -d;
-./ffc-sfd-customer-receiver-preferences/scripts/start -d;
-./ffc-sfd-proxy/scripts/start -d; # Must be after all web applications
+if [ "${sfd}" = "true" ]; then
+  ./ffc-sfd-data/scripts/start -d;
+  echo "Waiting for cosmosDB to start" # Cosmos is a dependency for receiver services
+  echo "Cosmos countdown:"
+  for ((i=90; i>=1; i--)); do
+    printf "\r$i"
+    sleep 1
+  done
```

Comment:
Is this not encapsulated in data service start up script?  Why does it need to be here too?

Reply:
I thought it might be helpful to see something so you knew why it was taking so long and also how long was left.

I looked at running the logs from the data service but there were too many.

Happy to remove it if you think its not required, maybe could have a simple log message instead.